### DISCUSSION TITLE
Longhorn is making a lot of API server calls

### DISCUSSION THREAD

**Discussion by serhiynovos** [longhorn/longhorn#8310] [Q&A] [answered]
Hi. I found issue that etcd size significantly increased and IO as well. Based on this discussion https://github.com/rancher/rke2/discussions/5680 we found that longhorn.io is making a lot of API calls which can cause this.  Is there any way I can investigate what exactly issue can be ?

**Comment by serhiynovos** [comment]
I guess it can be relate to this issue https://github.com/longhorn/longhorn/issues/8114

**Reply by serhiynovos** [reply]
If it's this issue, I installed longhorn from https://git.rancher.io/charts#release-v2.8 where latest version is 1.5.4. Is there are any safe way to update do 1.6.1 to try it out ?

**Reply by derekbit** [reply]
We will release a rancher chart for longhorn v1.6.1 soon. Please keep an eye on the ticket https://github.com/rancher/charts/pull/3754.

**Reply by serhiynovos** [reply]
It should not create any issues for migrating from 1.5.4 to 1.6.1 right now?

**Reply by derekbit** [reply, marked as answer]
Upgrade from 1.5.4 to v1.6.1 is fine.
Before upgrading your Longhorn, please check the [important notes](https://longhorn.io/docs/1.6.1/deploy/important-notes/) first. Thank you.
